src/v3/src/components/Widget/index.tsx

```diff
@@ -351,6 +352,7 @@ export const Widget: FunctionComponent<WidgetProps> = (widgetProps) => {
                 authCoinProps={buildAuthCoinProps(idxTransaction)}
               />
               <AuthContent>
+                <ConsentHeader />
```

Comment:
Created a tech debt ticket to handler all header elements separately: https://oktainc.atlassian.net/browse/OKTA-598852